<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAIN_MENU</class>
 <widget class="QWidget" name="MAIN_MENU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SCHEDULE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SCHEDULE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EXIT">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>700</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>信科教务重度依赖</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>960</width>
     <height>695</height>
    </rect>
   </property>
   <property name="text">
    <string>background_pic</string>
   </property>
  </widget>
  <widget class="QLabel" name="bottom_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="score">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>BEGIN</string>
   </property>
  </widget>
  <zorder>background_pic</zorder>
  <zorder>title_background</zorder>
  <zorder>SCHEDULE</zorder>
  <zorder>EXIT</zorder>
  <zorder>icon</zorder>
  <zorder>label</zorder>
  <zorder>bottom_background</zorder>
  <zorder>time</zorder>
  <zorder>score</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
